modules/test/playwright/tests/staging-configuration-web/stagingConfiguration.spec.ts

```diff
@@ -37,3 +60,84 @@ test('Check if local staging can be enabled', async ({
 
 	await stagingConfigurationPage.enableLocalStaging({});
 });
+
+testFlagsEnabled(
+	'Check if local staging with page-scoped Web Content can be enabled',
+	{tag: ['@LPS-83147']},
+	async ({
+		apiHelpers,
+		page,
+		pageEditorPage,
+		uiElementsPage,
+		webContentDisplayPage,
+	}) => {
+		const siteName = getRandomString();
+		const layoutName = getRandomString();
+		const webContentName = getRandomString();
+
+		const site = await apiHelpers.headlessSite.createSite({
+			name: siteName,
+		});
+
+		apiHelpers.data.push({id: site.id, type: 'site'});
```

Comment:
The test is failing in this line. Both in the ci and in my local. Could you check it? :)